distributed/distributed-schema.yaml

```
@@ -117,6 +117,26 @@ properties:
             description: |
               How frequently to balance worker loads
 
+          worker-saturation:
+            type: number
+            description: |
+              Controls how many root tasks are sent to workers (like a `readahead`).
+
+              `max(floor(worker-saturation * worker.nthreads), 1)` root tasks are sent to a
```

```suggestion
              Up to worker-saturation * nthreads root tasks are sent to a
```
I personally prefer readability over correctness